app/pages/lab/translations/index.jsx

```diff
@@ -58,7 +70,7 @@ class TranslationsManager extends React.Component {
         }
         <h2>Project language menu</h2>
         <p>
-          Check which languages you’d like to make available to your volunteers. If you don’t see a published language as a selection below please email <a href="mailto:contact@zooniverse.org">contact@zooniverse.org</a> with your project ID.
+          Use the checkboxes below to make a translation available to project volunteers. If you created a project translation that does not appear in the checkbox list, the reason is typically because that language does not currently have platform-level translations available. In this case, please email <a href="mailto:contact@zooniverse.org">contact@zooniverse.org</a> with your project ID to discuss next steps for adding a platform-level translations for that language.
```

Comment:
I'm likely mistaken, but not sure if should be "a platform-level translation" or omit "a" so it's "platform-level translations" .

Reply:
nice catch -- I'll tweak the text.